### DISCUSSION TITLE
H1 font size

### DISCUSSION THREAD

**Discussion by tmt514** [facebook/docusaurus#5356] [Q&A]
Hi, I was trying to change the h1 heading font size via `--ifm-h1-font-size` but realized that it is overwritten here:
https://github.com/facebook/docusaurus/blob/174fd2d248e9e5ffb200fa6d633113dfd0355c8b/packages/docusaurus-theme-classic/src/theme/Heading/styles.module.css#L13

Are there any suggested workaround for me to remove this line (from the custom css side)?
Thanks in advance!

**Comment by lex111** [comment]
At the moment we're not setting h1 styles correctly, so we'll fix that soon so you can change size of h1 via CSS var.

In the meantime, you can override the current font size in your custom CSS file something like that:

```css
[class^=h1Heading] {
    font-size: 2rem;
}
```

**Reply by tmt514** [reply]
Thank you so much!

**Reply by asksazz** [reply]
Is this issue fixed? The above suggested solution doesn't work for me. 
The H1 font size is too large for mobile devices. Is there a way to override it?
